File: lib/getData.js
import { createClient } from '@supabase/supabase-js'
import config from '../config.js'

const {
  SUPABASE_URL,
  SUPABASE_ANON_KEY
} = config

const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY)

export default async function getData () {
  const { data: groups, error } = await supabase
    .from('groups')
    .select('title, albums (title, year, is_highlighted, youtube, spotify, yes_no_wave, artists (name))')

  if (error) {
    throw error
  }

  return groups
}

File: lib/get_data.py
import sys
sys.path.append('../')
from config import config
from supabase import create_client, Client

supabase: Client = create_client(config["SUPABASE_URL"], config["SUPABASE_KEY"])

def get_data():
    response = supabase.table("groups").select("title, albums (title, year, is_highlighted, youtube, spotify, yes_no_wave, artists (name))").execute()
    print(response)
    return response
